debian/changelog

```diff
@@ -1,3 +1,11 @@
+cloud-init (24.4~3+really24.3.1-0ubuntu2) oracular; urgency=medium
```

Comment:
Would `24.4~4` make more sense here as this is devel?

Reply:
I was wondering the same, but this is an ubuntu downstream change just beyond the `24.4~3+really24.3.1` and into the development release still, so I'd think we would follow the [ubuntu-specific version bump in devel series](https://github.com/canonical/ubuntu-maintainers-handbook/blob/main/VersionStrings.md#version-adding-a-change-in-the-current-ubuntu-development-release).